backend/src/app.js

```diff
@@ -71,6 +72,25 @@ app.get('/api/v1/categories', async (_, res) => {
   res.json(categories());
 });
 
+app.get('/api/v1/legislators/search/offices', async(req, res, next) => {
```

Comment:
Is this creating our own API call? The URL string for openstates (after the host name) doesn't include /api/v1/legislators/search/offices